draft-ietf-tls-keylogfile.md

```diff
@@ -218,7 +255,8 @@ by any entity that is not otherwise authorized to observe or modify the content
 of connections for which secrets are logged could represent a privilege
 escalation attack.  Implementations that enable logging also need to ensure that
 access to logged secrets is limited, using appropriate file permissions or
-equivalent access control mechanisms.
+equivalent access control mechanisms. To minimize the risk of accidental activation
+in production, implementers SHOULD incorporate appropriate compile-time controls.
```

Comment:
```suggestion
equivalent access control mechanisms.
```

Separate pull request for this if you want to add it.  I don't think we need it either way, but I'm happy to discuss it.

Reply:
This was based on comments on the list from adoption call. Happy to drop it as I believe this kind of thing heavily depend on implementation details.

Reply:
Yeah, this is just unnecessary and a little weird.  We don't tell people how to structure their code in specifications.